cgroup/util.go

```diff
@@ -139,6 +147,14 @@ func SupportedSubsystems(rootfsMountpoint string) (map[string]struct{}, error) {
 		}
 
 		subsystem := fields[0]
+		if len(fields) >= 4 {
```

Comment:
I added a comment showing the format of the lines being parsed. That's probably sufficient to understand why it checks for `4`.

Each line has 4 tokens. I check to make sure there are at least that many tokens in the slice before dereferencing index=3. The check is "at least" because in the future someone my add a new column to the kernel output and I don't want this to break (it's happened in other parts of this code when testing on newer kernel versions).